src/ui/public/chrome/config/filter.html

```diff
@@ -1,7 +1,7 @@
 <kbn-timepicker
-  from="timefilter.time.from"
-  to="timefilter.time.to"
-  mode="timefilter.time.mode"
+  from="opts.timefilter.time.from"
```

Comment:
Don't like the way this works, this requires every controllers have `$scope.opts.timefilter = timefilter` to effectively act as a proxy property to the timefilter service.